src/app/services/core-event-handlers.service.ts

```diff
@@ -21,11 +22,13 @@ export class CoreEventHandlersService {
     private readonly workflow: WorkflowService,
     private readonly localData: LocalDatabaseService,
     private readonly adapterInstall: AdapterInstallService,
-    private readonly translate: TranslateService
+    private readonly translate: TranslateService,
+    private readonly settings: AppSettingsService
   ) {
+    const expose = this.settings.get('exposeCoreActions', false);
```

Comment:
The main settings page now has an option to "Expose core actions". If this is enabled, we allow workflows to access these.

Reply:
Wouldn't it be better if this comment was in the code? @the-greenman

Reply:
Comment added